### PR TITLE
✨ Added embed for Yandex Zen content-recommendation platform

### PR DESCRIPTION
**Pull request by Dannaz**
Added embed for widget of Yandex Zen content-recommendation platform

### CODE REVIEW

**Comment by Dannaz** [comment]
I signed it!

**Comment by Dannaz** [comment]
Done

**Comment by Dannaz** [comment]
Hello, is anything wrong in this PR? Can you review this?
P.S. Ping

**Comment by Dannaz** [comment]
Ping

**Comment by Dannaz** [comment]
Please, take a look at this PR.

**Comment by lannka** [comment]
pls fix:

```
ads/zen.js:30: ERROR - Parse error. Invalid trailing comma in arguments list
```

**Comment by Dannaz** [comment]
I fixed it.

**Comment by Dannaz** [comment]
Ping. I fixed error

File: ads/zen.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        }
+      },
+      failCallback: () => {
+        if (typeof data.successCallback === 'function') {
+          data.failCallback();
+        }
+      },
+    });
+
+    global.YandexZen.renderWidget(config);
```
Comment (lannka) [inline ads/zen.js]: this line is failing the travis test.

you will need to add ads.extern.js to tell compiler to not minify the string.

**Comment by Dannaz** [comment]
@lannka , I added external properties used by my code to `ads.extern.js`
If I correctly understand comment in `ads.extern.js`, I should not define there internal props, like object fields. Am I right?

**Comment by Dannaz** [comment]
Ping.

**Comment by Dannaz** [comment]
Hello, is anything wrong in this PR? Can you give me some info about it?
P.S. Ping

**Comment by Dannaz** [comment]
@aghassemi , @lannka ping

**Comment by lannka** [comment]
Sorry for the delay. Thanks for the PR. It looks good. Merged.

**Pull request merged**